### ISSUE TITLE
Eliminate NaN insights

### ISSUE THREAD

**Issue by sgoggins** [chaoss/augur#623]
**Is your feature request related to a problem? If so, please describe the problem:**
The insight worker detects anomalies when "NaN" values are generated. Can these be filtered out? 
```
New Augur Insight
There were 0 new_contributors on https://github.com/ropensci/ecoengine over the last 90 days.
That represents a NaN% increase from the mean!
12:26
New Augur Insight
There were 0 issues on https://github.com/ropensci/ecoengine over the last 90 days.
That represents a NaN% increase from the mean!
12:26
New Augur Insight
There were 0 issues on https://github.com/ropensci/ecoengine over the last 90 days.
That represents a NaN% increase from the mean!
12:26
New Augur Insight
There were 0 issues on https://github.com/ropensci/ecoengine over the last 90 days.
That represents a NaN% increase from the mean!
12:26
New Augur Insight
There were 0 issues on https://github.com/ropensci/ecoengine over the last 90 days.
That represents a NaN% increase from the mean!
```

**Comment by gabe-heim** [comment]
I already have a fix on my local machine, I just have not pushed yet. I will close this when I push.

**Comment by gabe-heim** [comment]
Fixed in 41f4c158fbcaddc12da7a053c803bac0edeb8e10
